port/common/omrfile.c

```diff
@@ -630,7 +630,7 @@ omrfile_vprintf(struct OMRPortLibrary *portLibrary, intptr_t fd, const char *for
 	COPY_VA_LIST(copyOfArgs, args);
 
 	/* What is size of buffer required ? Does not include the \0 */
```

Comment:
I thought the API did return the size including the `\0`?  Can you verify whether or not it does?  Thanks